whois-commons/src/main/resources/nrtm_data.sql

```diff
@@ -0,0 +1,2 @@
+insert into source(id, name)
+values (100, 'RIPE_TEST_SOURCE');
```

Comment:
use the existing source name `TEST` or the schema name `WHOIS_TEST_TEST`

Reply:
this has since been removed -- data used in the tests is set up explicitly in the test class